src/Components/Title.jsx

```diff
@@ -1,8 +1,12 @@
-export default function Title(){
-  return(
-    <div>
-      <h1>Recipe Finder</h1>
-      <p>A tool to help you use up those leftover ingredients</p>
-    </div>
-  )
+import "./Title.css";
+
+export default function Title() {
```

Comment:
Revert to original

Reply:
Hi Roy, I have made the suggested changes. Thank you

Reply:
Please revert the work done in this Title component work. Changes to this file should not appear in this PR.